common/api/core-bentley.api.md

```diff
@@ -1558,6 +1618,25 @@ export class TransientIdSequence {
     get next(): Id64String;
 }
 
+// @public
+export class TupleKeyedMap<K extends readonly any[], V> implements PartialMap<K, V> {
```

Comment:
TupleKeyedMap is exported but implements non-exported PartialMap interface.

Reply:
I am choosing to remove `PartialMap`, it was part of an earlier iteration of the implementation.

Reply:
removed